french/dessine/porte.md

```diff
@@ -1,10 +1,11 @@
 J'ouvre la porte. J'entre dans une pièce où se trouvent d'autres portes.
 La porte qui m'a attire le plus est la porte dessinee avec du miel. Elle brillait!
-Donc je decide de
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.